octobot_trading/personal_data/orders/order_util.py

```diff
@@ -95,7 +96,9 @@ def check_cost(total_order_price, min_cost):
     return True
 
 
-async def get_pre_order_data(exchange_manager, symbol: str, timeout: int = None):
+async def get_pre_order_data(exchange_manager, symbol: str, timeout: int = None,
+                             portfolio_type=commons_constants.PORTFOLIO_AVAILABLE,
```

Comment:
We shouldn't use `portfolio_type` anymore. It doesn't make sense with Assets. We should migrate portfolio_type usage instead of adapting methods

Reply:
what do you see we should do in this context ?

Reply:
I think we should return the asset instance instead of asking for a `portfolio_type`.

Reply:
In this case, we will have to call `market_quantity = market_available / mark_price if mark_price else constants.ZERO` every time we need it after calling `get_pre_order_data` if we don't directly manipulate portfolio values but just return assets, I'm not sure it's better.

Reply:
I think that we will have to think about the role of this method. It becomes a bit of a method where you put everything into it.  This problem would probably not be discussed with a method with a more precise scope in my opinion.